cpp/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 cmake_minimum_required (VERSION 2.8)
```

Comment:
We could make gym_tcp a dependency, and link against them. I think there is no need to copy the files, since the gym repo is kinda independent.